pkgs/development/python-modules/bacpypes/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildPythonPackage, fetchPypi, fetchFromGitHub
+, wheel, pytest, pytestrunner }:
+
+buildPythonPackage rec {
+  version = "0.17.7";
+  pname = "bacpypes";
+
+  src = fetchFromGitHub {
+    owner = "JoelBender";
+    repo = "bacpypes";
+    rev = "v${version}";
+    sha256 = "1bb630xpdachl3ij5bvxa67qybgrd2lqycqk4yicypyf5gs659j6";
+  };
+
+  checkInputs = [ pytest ];
+  nativeBuildInputs = [ pytestrunner ];
+  propagatedBuildInputs = [ wheel ];
+
+
+  # Time test fails
+  doCheck = false;
```

Comment:
As the tests are (currently) disabled, the testing frameworks should be commented out in the input-lists since they're not needed atm.

Reply:
or better yet, since you're using pytest. Just exclude the ones that don't work in a the sandbox environment. If they are a legitimate failure due to dependency issues, then that means they are in an invalid state.

Reply:
How can I disable them from nix?

Reply:
using pytest, you can do 
```
    pytest path/to/test/dir --ignore=path/to/test/ignore.py -k 'not some_wierd_test_name'
```

Reply:
@jonringer I was able to disable the failing time test by switching to the `pytestCheckHook` and using `disabledTests`